apiserver/params/params.go

```diff
@@ -768,37 +768,97 @@ type RebootActionResult struct {
 type Life multiwatcher.Life
 
 const (
-	Alive = Life(multiwatcher.Alive)
-	Dying = Life(multiwatcher.Dying)
-	Dead  = Life(multiwatcher.Dead)
+	Alive Life = "alive"
+	Dying Life = "dying"
+	Dead  Life = "dead"
 )
 
 // Status represents the status of an entity.
 // It could be a unit, machine or its agent.
 type Status multiwatcher.Status
 
+// TranslateLegacyStatus returns the status value clients expect to see for Juju 1.x.
+func TranslateLegacyStatus(in Status) Status {
+	switch in {
+	case StatusFailed:
+		return StatusDown
+	case StatusActive:
+		return StatusStarted
+	default:
```

Comment:
Fair question - the current understanding is that scripts are almost exclusively written to poll for a status of Started (and also Down). Stopped/Stopping is a transitional phase during unit destruction. It's similar to why there's no translation for Pending/Allocating